k-distribution/include/kframework/builtin/domains.md

```diff
@@ -2938,6 +2967,7 @@ You can:
 
 * Compute the bitwise complement `~MInt` of an `MInt`.
 * Compute the unary negation `--MInt` of an `MInt`.
+* Compute the power `^MInt` of two `MInt`s. Currently. Currently, only 64 and 256-bits is supported.
```

Comment:
you should probably clarify that the machine integers are treated as unsigned for the purposes of this operation.

Reply:
Done! Thanks!